Kernel/Net/NetworkAdapter.h

```diff
@@ -50,7 +50,7 @@ class NetworkAdapter : public RefCounted<NetworkAdapter>
 
     virtual StringView class_name() const = 0;
 
-    const String& name() const { return m_name; }
+    const StringView name() const { return m_name->view(); }
```

Comment:
Isn't that supposed to be a `const StringView&` return value? what is the meaning of `const StringView`?

Reply:
it's supposed to just be `StringView`, KString::view returns a new StringView, so this can't be a reference.